nyx_client/README.md

```diff
@@ -80,6 +80,29 @@ client.create_data(
 )
 ```
 
+### I want to share data using the instance's storage to upload
+
+```python
+from nyx_client import NyxClient
+
+client = NyxClient()
+
+with open("somefile.csv") as f:
+  client.create_data(
+      name="MyData1",
+      title="My Data #1",
+      description="The description of the data #1",
+      size=1080,
```

Comment:
Size should not need to be specified?

Reply:
I just tested this:
1. When creating - the returned `Data.size` was what I set it to (as opposed to real size)
2. When using `NyxClient.get_my_data_by_name`, the returned size was zero.